pontoon/teams/static/js/bugzilla.js

```diff
@@ -65,6 +65,16 @@ var Pontoon = (function (my) {
                   html: formatter.format(new Date(bug.creation_time))
                 }).appendTo(tr);
 
+                $('<td>', {
+                  class: 'assignee',
+                  html: formatter.format(new Date(bug.assignee))
```

Comment:
`bug.assignee` is not a date, so this will result in a JS. Also, I think the field name is [`assigned_to`](https://wiki.mozilla.org/Bugzilla:REST_API).

Reply:
Okay, I can't believe I wrote that.

Reply:
I will do well to test the code and be more careful. Thanks

Reply:
Okay I can see that 'assigned_to' on the linked location.  

So that leaves us with only the 'assigned_to' field; after taking out the 'status'.